github-profiles/lmendev-2708/lmendev-2708.md

```diff
@@ -0,0 +1,39 @@
+https://github.com/Lmendev
```

Comment:
[lmendev](https://github.com/lmendev)

a wise man once told me, you should change this to make a link to your profile ;)

Reply:
Oh!! 😢 this is outdated, it was corrected in https://github.com/undefined-academy/semana-1/pull/473/commits/076c8689bdaebec604f1f68fc4f14c22e70c02cf 8 hours ago :)